main.py

```diff
@@ -44,17 +46,23 @@ def get_stats():
         fmt_percent = format(l['percent'], '0.2f').zfill(5) # to provide a neat finish.
         data_list.append(
             f"{l['name']}{' '*(12-ln)}{l['text']}{' '*(20-ln_text)}{make_graph(l['percent'])}   {fmt_percent} %")
+    print("Graph Generated")
     data = ' \n'.join(data_list)
-    return '```text\n'+this_week()+'\n\n'+data+'\n```'
+    if show_title == 'false':
+        print("Stats with Weeks in Title Generated")
+        return '```text\n'+this_week()+'\n\n'+data+'\n```'
+    else:
+        print("Usual Stats Generated")
+        return '```text\n'+data+'\n```'
 
 
-def decode_readme(data: str):
+def decode_readme(data: str) -> str:
     '''Decode the contets of old readme'''
     decoded_bytes = base64.b64decode(data)
     return str(decoded_bytes, 'utf-8')
 
 
-def generate_new_readme(stats: str, readme: str):
+def generate_new_readme(stats: str, readme: str) ->str:
```

Comment:
What is this `-> str`? return type?

Reply:
Cool! I didn't know [this](https://stackoverflow.com/questions/14379753) feature existed!

Reply:
Yea me too. Good to know.

Reply:
They're function Annotations. Not exactly return type but serves the purpose